sdk/nodejs/src/eventQueue.ts

```diff
@@ -106,10 +106,16 @@ export class EventQueue {
             this.eventFlushIntervalMS,
         )
 
+        const eventQueueOptions = {
+            eventRequestChunkSize: chunkSize,
```

Comment:
how did this work before if you're renaming this property? or did it just not work

Reply:
it unwrapped it from the options as a different name `chunkSize`, the original property name on the options is `eventRequestChunkSize`

Reply:
although this does technically change it so that the chunk size is now defaulted to 100 defined above, not sure if that's what we want